index.html

```diff
@@ -1,64 +1,131 @@
+                var numberTwo = document.forms.inputs.numberTwo.value;
+
+                // Do our calculation..
+                document.forms.inputs.result.value = parseInt(numberOne) / parseInt(numberTwo);
+            }
+        };
+    </script>
 </head>
 
 <body>
-    <div>Header</div>
-    <div>
-       <!--
-         - allow two numbers to be entered
-         - show buttons for add and subtract
-         - show the result
-        -->
-        <form name="inputs">
-            <div>
-                Number 1:
-                <input type="number" name="numberOne" class="form-control" placeholder="First number" aria-label="Number1" >
-            </div>
-            <div>
-                Number 2:
-                <input type="number" class="form-control" name="numberTwo">
-            </div>
-            <div>
-                <button type="button" class="btn btn-primary" onclick="calculator.addNumbers()" >Add</button>
-                <button type="button" class="btn btn-primary">Substract</button>
-            </div>
-            <div>
-                <input name="result" value="0" class="form-control" disabled />
-            </div>
-        </form>
 
+    <!-- Visible header at top of page using Bootstrap -->
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+        <a class="navbar-brand" href="#">Hollis's JS Calculator</a>
+        </div>
+    </nav>
+    <br></br>
 
-    </div>
-    <div>Footer</div>
+    <!-- Allow two numbers to be entered
+show buttons for add and subtract
+        then show the results  -->
+    <!-- Create a form to reference in JS -->
+    <form name="inputs">
+        <!-- Create our card to hold our calculator controls and center it -->
+        <div class="card bg-light mb-3" style="max-width: 90%;margin:0 auto;">
+            <h4 class="card-header">Main Calculator</h4>
+            <!-- Input box one-->
+            <div class="input-group mb-3" style="max-width: 95%;margin:0 auto;padding-top: 15px;">
+                <div class="input-group-prepend">
+                    <span class="input-group-text" id="basic-addon1">Number One:</span>
+                </div>
+                <input type="number" pattern="[0-9]*" class="form-control" placeholder="" aria-label="Number One"
+                    aria-describedby="basic-addon1" name="numberOne"></input>
+            </div>
 
+            <!-- Input box two-->
+            <div class="input-group mb-3" style="max-width: 95%;margin:0 auto;">
+                <div class="input-group-prepend">
+                    <span class="input-group-text" id="basic-addon1">Number Two:</span>
+                </div>
+                <input type="number" pattern="[0-9]*" class="form-control" placeholder="" aria-label="Number Two"
+                    aria-describedby="basic-addon1" name="numberTwo"></input>
+            </div>
 
+            
+                <!-- Create the function buttons -->
```

Comment:
inline style is ok here for `style="padding-bottom: 15px;margin:0 auto;"` but could be set in the header style tag instead.

For rough example:
```
<style>
.calc-btn-group {
  padding-bottom: 15px;
  margin:0 auto;
}
</style>
```

And then your tag can be:

```
<div class="btn-group calc-btn-group" role="Group">
```